sql-to-dbsp-compiler/.calcite_version

```diff
@@ -0,0 +1,7 @@
+CALCITE_BUILD_NEXT="y"
```

Comment:
I don't like hidden files, you never find them
why can't this be a regular shell script with a normal name?